dnfdaemon-server/services/repo/repo.cpp

```diff
@@ -49,7 +49,7 @@ uint64_t repo_size(libdnf::rpm::SolvSack & sack, const libdnf::WeakPtr<libdnf::r
     uint64_t size = 0;
     libdnf::rpm::SolvQuery query(&sack);
     std::vector<std::string> reponames = {repo->get_id()};
-    auto pkgset = query.ifilter_repoid(libdnf::sack::QueryCmp::EQ, reponames).get_package_set();
+    auto pkgset = query.ifilter_repoid(libdnf::sack::QueryCmp::EQ, reponames);
```

Comment:
Why do you want to create an additional reference? What about to use query directly for iteration?
```
for (auto pkg : query) {
```

Reply:
Yes. Good point. I overlooked it.

Reply:
Removed.